### DISCUSSION TITLE
We do not need `@types/jsforce` with v3 anymore right?

### DISCUSSION THREAD

**Discussion by pavitra-infocusp** [jsforce/jsforce#1552] [Q&A] [answered]
https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/jsforce

**Comment by cristiand391** [comment, marked as answer]
correct, v3 includes the types in the published npm package, see:
https://github.com/jsforce/jsforce/blob/main/MIGRATING_V1-V3.md#typescript-type-definitions
